### PR TITLE
[fr] Finish translating core-libraries/text

### PR DESCRIPTION
**Pull request by HavokInspiration**

### CODE REVIEW

File: fr/core-libraries/text.rst [hunk 1]
```diff
@@ -43,40 +43,49 @@ Convertion des Chaînes de Caractères en ASCII
...
-pass using the ``$transliteratorId`` argument or change the default identifier
-string using ``Text::setTransliteratorId()``. ICU transliteration identifiers
-are basically of form ``<source script>:<target script>`` and you can specify
-multiple conversion pairs separated by ``;``. You can find more info about
-transliterator identifiers
-`here <http://userguide.icu-project.org/transforms/general#TOC-Transliterator-Identifiers>`_::
-
-    // apple puree
+La méthode ``translisterate`` convertit par défaut tous les caractères dans la
+chaîne soumises dans son équivalent en caractères ASCII. La méthode s'attend à
```
Comment (antograssiot) [inline fr/core-libraries/text.rst]: soumise

File: fr/core-libraries/text.rst [hunk 2]
```diff
@@ -151,14 +160,14 @@ Vous pouvez utiliser les options suivantes dans le tableau options::
...
-=============
+Fixer la largeur d'un texte
+===========================
 
 .. php:staticmethod:: wrap($text, $options = [])
 
 Entoure un block de texte pour un ensemble de largeur, et indente aussi les
-blocks. Peut entourer intelligemment le texte ainsi les mots ne sont pas sliced
-across lines::
+blocks. Peut entourer intelligemment le texte ainsi les mots ne sont pas couper
```
Comment (antograssiot) [inline fr/core-libraries/text.rst]: coupés

File: fr/core-libraries/text.rst [hunk 3]
```diff
@@ -43,40 +43,49 @@ Convertion des Chaînes de Caractères en ASCII
...
-    // apple puree
+La méthode ``translisterate`` convertit par défaut tous les caractères dans la
+chaîne soumises dans son équivalent en caractères ASCII. La méthode s'attend à
+avoir une chaîne encodée en UTF-8 en entrée. La conversion des caractères peut
+être controllée en utilisant des identifiants de translitération que vous
+passez via l'argument ``$transliteratorId`` de la méthode ou en changeant
+l'identifiant par défaut à l'aide de ``Text::setTransliteratorId()``.
+Les identifiants de translitération d'ICU sont généralement sous la forme
+``<source script>:<target script>`` et vous pouvez spécifier plusieurs couples
+ de conversion en les séparant par des ``;``. Vous trouverez plus
```
Comment (antograssiot) [inline fr/core-libraries/text.rst]: you need to trim the first space on this line. It cause the build to fail.

**Comment by HavokInspiration** [comment]
I just pushed a fix based on @antograssiot's comments.

**Comment by antograssiot** [comment]
:+1:

**Pull request merged**